src/api/config/initializers/airbrake.rb

```diff
@@ -107,6 +107,9 @@ def ignore_by_class?(notice)
 end
 
 def ignore_exception?(notice)
+  notice[:errors].pluck(:type).each do |type|
```

Comment:
It's a bit weird to have this code here. I suggest putting it in the block of `Airbrake.add_filter`, right above `notice.ignore! if ignore_exception?(notice)`.